README.md

```diff
@@ -991,7 +991,8 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :video_camera: [Frontend Interview Series](https://youtube.com/playlist?list=PL-Jc9J83PIiFXzRwSTSwqL7U2hoMk6K6F)
 - :video_camera: [Frontend Interview Series](https://youtube.com/playlist?list=PL-Jc9J83PIiFXzRwSTSwqL7U2hoMk6K6F)
 - :video_camera: [Frontend with HTML CSS JS & Browser APIs](https://youtube.com/playlist?list=PL-Jc9J83PIiHU9RkY9sfh3G64-bd0ptvC)
-
+- :other: [Frontend Coding Challenges](https://www.frontendmentor.io)
+- :other: [Frontend Coding Challenges](https://www.codewell.cc)
```

Comment:
Please change the title to the page's title.

Also, please add a new empty line after this.